src/main/kotlin/baseball/Baseball.kt

```diff
@@ -0,0 +1,60 @@
+package baseball
+
+import baseball.utils.Utils
+import camp.nextstep.edu.missionutils.Console
+
+class Baseball {
+    fun run(computer: List<Int>) {
+        while (true) {
+            val user = getPlayerNumber()
+            val strike = countStrike(computer, user)
+            val ball = countBall(computer, user)
+
+            printGameResult(strike, ball)
+            if(strike == INPUT_SIZE) break
+        }
+    }
+
+    fun getPlayerNumber() : List<Int> {
+        print("숫자를 입력해주세요 : ")
+        val userNumber = Console.readLine()
+        return Utils().checkInput(userNumber)
+    }
+
+    fun countStrike(computer: List<Int>, user: List<Int>) : Int {
+        var count = 0
+        for (i in 0..INPUT_SIZE-1) {
+            if (computer[i] == user[i]) count++
+        }
+        return count
+    }
+
+    fun countBall(computer: List<Int>, user: List<Int>) : Int {
+        var count = 0
+        for (i in 0..INPUT_SIZE-1) {
+            if(checkBallNumberSame(computer, user, i)) count++
+        }
+        return count
+    }
+
+    fun checkBallNumberSame(computer: List<Int>, user: List<Int>, index: Int) : Boolean {
+        for (i in 0..INPUT_SIZE-1) {
+            if(index == i) continue
+            if(computer[index] == user[i]) return true
+        }
+        return false
+    }
```

Comment:
직접 for문을 도는 것보단 contains함수를 활용하는 편이 깔끔해 보입니다.
```suggestion
       return conputer.contains(user[i]) 
```